app/posts/ecf-v2/2024-07-15-workshop-process.md

```diff
@@ -0,0 +1,69 @@
+---
+title: The process behind our problem prioritisation and ideation workshops
+description: "Our journey of prioritising problems and developing high level solutions to take forward"
+date: 2024-07-15
+---
+
+Over the last month our team ran several workshops with the aim of consolidating our top problems across our 4 user groups and prioritising them followed by ideating on solutions on how to address our biggest problems. This consisted of 4-5 problems workshops and 1-2 ideation workshops.
+
+### How we approached the problem prioritisation exercise
+
+#### Identified the problems to focus on by looking at:
+
```

Comment:
```suggestion
### Identified the problems to focus on

We achieved this by looking at:
```

Reply:
I think subheadings wouldn't normally be followed by bullets, so just making the following suggestions :)